source/models/the-fixture-adapter.md

```diff
@@ -79,6 +79,6 @@ a uniquely identifiable field. By default, Ember Data assumes this key
 is called `id`. Should you not provide an `id` field in your fixtures, or
 not override the primary key, the Fixture Adapter will throw an error.
 
-[1]: /guides/models/defining-models
```

Comment:
I'm unsure about these; I don't know if there is a root-template for these links that automatically matches the selected version, but those broken links are live.